cogs/utility.py

```diff
@@ -265,30 +267,60 @@ def format_not_found(self, ctx, command):
             else:
                 em.description = 'Already up to date with master repository.'
 
-            
         await ctx.send(embed=em)
 
-    @commands.command(name='status', aliases=['customstatus', 'presence'])
+    @commands.command(aliases=['presence'])
     @commands.has_permissions(administrator=True)
-    async def _status(self, ctx, *, message):
-        """Set a custom playing status for the bot.
-
-        Set the message to `clear` if you want to remove the playing status.
+    async def activity(self, ctx, activity_type: str, *, message: str = ''):
```

Comment:
Why not remove the message default value and let the command handler output the usage of the command.

Reply:
The default's for the `clear` activity type to remove the current activity, I don't think `clear` will work if the default's removed.

Reply:
Yes, the command error handler checks if that error was raised and sends the help message for that command.

Reply:
Updated.